scripts/deployment/restart_all_servers.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o xtrace
+
+# https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad
+
+# This script is run to restart the Docker services on ALL Open Library hosts.
+
+RESTART_THIS_SERVER="/opt/openlibrary/script/restart_this_server.sh"
```

Comment:
Use it directly; makes it easier to read than having it in an environment variable.